pkg/controller/deployer.go

```diff
@@ -125,6 +128,25 @@ func (d *deployer) Undeploy(function *v1.Function) error {
 		&metav1.DeleteOptions{PropagationPolicy: &propagation})
 }
 
+func (d *deployer) Update(function *v1.Function) error {
+	deployment := v1beta1.Deployment{
```

Comment:
I had a working version that did a get + update.
As a matter of fact, I'm not even sure we want/need a new `Update` function. `Deploy` with an `if` should be fine

Reply:
I'd prefer to _mirror_ the structure of the k8s API and have separate `Deploy` and `Update` operations rather than overload functions to do multiple things.

Reply:
OK, fair enough. I think the no of replicas should not be set to zero then, at the very least